src/dlgProfilePreferences.cpp

```diff
@@ -361,6 +363,18 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pH) : QDialog(pF
         } else {
             checkBox_showDefaultArea->hide();
         }
+
+        comboBox_encoding->addItem(QLatin1String("ASCII"));
+        comboBox_encoding->addItems(pHost->mTelnet.getEncodingsList());
```

Comment:
Could UTF-8 follow ASCII, since it'll likely be a popular option in the future given it's popularity already?